tests/schema/tests.py

```diff
@@ -1671,6 +1730,60 @@ class Meta:
         with self.assertRaises(IntegrityError):
             Tag.objects.create(title='bar', slug='foo')
 
+    def test_remove_field_uniqueness_does_not_remove_custom_constraints(self):
+        # Create table
+        with connection.schema_editor() as editor:
+            editor.create_model(AuthorWithUniqueName)
+
```

Comment:
You can omit the blank line before each comment.

Reply:
removed